src/process-replies.js

```diff
@@ -6,7 +6,7 @@ const valueMap = {
   '3': 0,
   '4': 0.8,
   '5': 1,
-  '6': null
+  '6': null,
```

Comment:
I guess this is prettier running in your editor making these changes. Maybe we should add prettier to the project and run it in CI so we don't have unrelated changes like this in the PRs?

Reply:
Great point. I will do that in a separate PR

Reply:
https://github.com/kristjanmik/kjosturett-web/pull/69